src/mcts/stoppers/stoppers.h

```diff
@@ -124,11 +124,12 @@ class KldGainStopper : public SearchStopper {
 // best move to potentially become the best one, stop the search.
 class SmartPruningStopper : public SearchStopper {
  public:
-  SmartPruningStopper(float smart_pruning_factor);
+  SmartPruningStopper(float smart_pruning_factor, int64_t minimum_batches_);
```

Comment:
Trailing underscore in parameter name (not needed).

Reply:
oops. :)